Sources/PointFree/Account/Cancel.swift

```diff
@@ -262,7 +262,7 @@ private let formRowView = View<Stripe.Subscription> { subscription in
 // MARK: - Emails
 
 private func sendCancelEmail(to owner: Database.User, for subscription: Stripe.Subscription)
-  -> EitherIO<Prelude.Unit, SendEmailResponse> {
+  -> EitherIO<Prelude.Unit, Mailgun.SendEmailResponse> {
```

Comment:
I created a property `Mailgun` type for holding the endpoints we hit like we do for `GitHub` etc, and so moved this model type into there.